src/main/java/Ui/Ui.java

```diff
@@ -0,0 +1,117 @@
+package Ui;
+
+import Tasks.Task;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Ui {
+    private static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+    private static final String WELCOME_MESSAGE = "Hello from\n" + LOGO;
+    private static final String DIVIDER_LINE = "_______";
+
+    /**
+     * Prints the welcome message.
+     */
+    public void showWelcome() {
+        System.out.println(WELCOME_MESSAGE);
+    }
+
+    /**
+     * Scans for user input and returns the result as a String.
+     * @return Command inputted as a String
+     */
+    public String readCommand() {
+        return new Scanner(System.in).nextLine();
+    }
+
+    /**
+     * Prints a message once a task has been added.
+     * @param numOfTasks total number of tasks after a task has been added
+     * @param taskDescription description of the newly added task
+     */
+    public void showAddTaskMsg(int numOfTasks, String taskDescription) {
+        System.out.println("Got it. I've added this task:\n  " + taskDescription);
+        if (numOfTasks == 1) {
+            System.out.println("Now you have " + numOfTasks + " task in the list.");
```

Comment:
Instead of using a + operator to concatenate your strings, you can consider using the String.format() method to attain the correct format.